iothub/device/devdoc/Convention-based operations.md

```diff
@@ -69,13 +75,22 @@ public abstract class PayloadCollection : IEnumerable, IEnumerable<object> {
 }
 
 public static class ConventionBasedConstants {
+    public const char ComponentLevelCommandSeparator = '*';
     public const string AckCodePropertyName = "ac";
     public const string AckDescriptionPropertyName = "ad";
     public const string AckVersionPropertyName = "av";
     public const string ComponentIdentifierKey = "__t";
     public const string ComponentIdentifierValue = "c";
     public const string ValuePropertyName = "value";
 }
+
+public class StatusCodes {
```

Comment:
You'll probably want a more unique name for this. How about `IotStatusCodes`? Or maybe `DeviceStatusCodes`?

Reply:
I like `IotStatusCodes`, only thing is, would people think that this is a status code that IoT Hub sends?
Since these status codes are sent by the client, how about `ClientStatusCodes`?

Reply:
`CommonClientStatusCodes` ?

Reply:
I wonder if `CommonClientStatusCodes` would make people think that these are common status codes that represent the status of a client? Should we call it `CommonClientResponseCodes`?